packages/block-editor/src/components/index.native.js

```diff
@@ -50,7 +50,10 @@ export { default as BlockToolbar } from './block-toolbar';
 export { default as DefaultBlockAppender } from './default-block-appender';
 export { default as __unstableEditorStyles } from './editor-styles';
 export { default as Inserter } from './inserter';
-export { Block as __experimentalBlock } from './block-list/block-wrapper';
+export {
+	Block as __experimentalBlock,
+	useBlockProps as __experimentalUseBlockProps,
+} from './block-list/block-wrapper';
```

Comment:
Thank you @ellatrix for updating the `native.js` side of this!

It seems that `useBlockProps` was defined in `block-list/block-wrapper.js `, but we have a `block-wrapper.native.js` version of `block-wrapper` without `useBlockProps`. 

Without going deep into the native errors, this seems to be at least part of the problem.